cfpb/consumerfinance.gov
# Update to design-system 0.34.0

Changelog
https://github.com/cfpb/design-system/blob/main/CHANGELOG.md#v0340-2023-12-05

Full dump of changes 
https://github.com/cfpb/design-system/compare/v0.33.0...v0.34.0


## Changes

- Update to design-system 0.34.0


## How to test this PR

1. PR checks should pass.
1. Visit http://localhost:8000/blog and confirm multiselects are working as expected.
